.gitignore

```diff
@@ -1,7 +1,11 @@
 /.idea/
 /docs/_build
 /docs/api
-**/.vscode/
+**/.vscode
+!.vscode/settings.json
+!.vscode/tasks.json
+!.vscode/launch.json
+!.vscode/extensions.json
```

Comment:
These were added by @edwardalee [in this commit](https://github.com/lf-lang/reactor-c/pull/224/commits/533002c66ef4fa77cb3cde8151091b85106b07d5).
Maybe to help with debugging in vs code?

Reply:
Oops, I think I only meant to change `/.vscode` to `**/.vscode`. There must have been a monkey bouncing on my keyboard to add lines 5-8.  Couldn't have been me because I didn't even know about the `!` syntax. :-)

Reply:
```suggestion
```